docs/modules/ROOT/pages/how-tos/configuring/customizing-the-build.adoc

```diff
@@ -97,44 +96,14 @@ NOTE: This example shows a separate Pipeline file, but your pipeline may also be
     the PipelineRun.
 
 
-== Extending the build pipeline with your own tasks
-
-Before you extend your build pipeline, be aware that doing so may cause builds to fail an link:https://enterprisecontract.dev/docs[Enterprise Contract] check, if you are using EC. To resolve this issue, please reference the xref:./customizing-the-build.adoc#preventing-issues-with-the-enterprise-contract[next procedure].
+== Adding Slack notifications
 
 .Procedure
 
-. In each Pipeline definition in the `.tekton` directory, add a new task to the `tasks` section.
-
-+
-Example task:
-+
-[source,yaml]
---
-  name: example-task
-  params:
-  - name: example-param
-    value: "Example"
-  runAfter:
-  - build-container  # You can be more specific by choosing another task
-  taskRef:
-    params:
-    - name: name
-      value: example-task  # metadata.name field of the Task
-    - name: bundle
-      value: quay.io/tekton-bundle-catalog/example-task-bundle:1.0
-      # For more details on tekton bundles, refer to https://tekton.dev/docs/pipelines/pipelines/#tekton-bundles
-    - name: kind
-      value: task
-    resolver: bundles
```

Comment:
This information is relevant for remote resolvers (along with the undocumented git resolver). Should we include some information/reference to these?